src/main/java/org/opensearch/plugin/insights/core/exporter/QueryInsightsExporterFactory.java

```diff
@@ -73,16 +72,16 @@ public void validateExporterConfig(final Settings settings) throws IllegalArgume
         }
         switch (type) {
             case LOCAL_INDEX:
-                final String indexPattern = settings.get(EXPORT_INDEX, DEFAULT_TOP_N_QUERIES_INDEX_PATTERN);
-                if (indexPattern.length() == 0) {
-                    throw new IllegalArgumentException("Empty index pattern configured for the exporter");
-                }
                 try {
-                    DateTimeFormat.forPattern(indexPattern);
+                    DateTimeFormat.forPattern(DEFAULT_TOP_N_QUERIES_INDEX_PATTERN);
                 } catch (Exception e) {
```

Comment:
Do we even need to validate this if we are always using defaults?

Reply:
I was wondering about that too. Let's remove it

Reply:
Removed validation from exporter and reader